webpack/assets/javascripts/react_app/common/hooks/API/APIHooks.js

```diff
@@ -18,6 +18,7 @@ import { APIActions } from '../../../redux/API';
 export const useAPI = (method, url, options) => {
```

Comment:
something like:
```suggestion
export const useAPI = (method, url, {key, asMutable = false, ...APIOptions}) => {
```

Reply:
this is similar to my suggestion  - `split the options object`  section :)